content/blog/slope-track-update-2021/slope-track-update-2021.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Suva Slope Track-App Update 2021
+slug: suva-slope-track-app-update-2021
+templateKey: blog-post
+image: slope-track-2021.jpg
+date: 2021-01-12
+author: Philip Schönholzer
+description: >-
+  Für 2021 haben wir einige schöne Verbesserungen und Erweiterungen an der Slope Track App von Suva vornehmen dürfen.
+categories:
+  - Allgemein
+---
+
+Mit dem Update für 2021 der Slope Track App für die Suva bieten wir einige interessante Optimierungen an. So verbessern wir unter anderem die Navigation und Darstellung der aufgezeichneten Abfahrten, erkennen die Abfahrten präzisier und bieten neue Tipps an.
+
+## Bessere Navigation und Darstellung von Abfahrten
+
+Die Abfahrten wurden bisher in einem begrenzten Bereich dargestellt. Zudem waren die Navigations-Elemente und dargestellten Werte zum Track etwas weit auseinander. Neu werden die Navigation und die relevanten Werte näher beieinander platziert. So ist die Navigation einfacher und das auslesen der Werte bequemer.
+
+Zudem kann die Karte nun auch im Vollbild dargestellt werden. In dieser Ansicht nutzen wir den gesamten Bildschirm um den Track darzustellen. Ausserdem ist es in dieser Ansicht möglich, die Karte zu in eine beliebige Richtung zu ziehen, um den gewünschten Bereich anzuzeigen.
```

Comment:
In dieser Ansicht nutzen wir den gesamten Bildschirm, um den Track darzustellen.
Ausserdem ist es in dieser Ansicht möglich, die Karte in eine beliebige Richtung zu ziehen, um den gewünschten Bereich anzuzeigen.